docs/reference/feature-servers/python-feature-server.md

```diff
@@ -200,6 +200,26 @@ requests.post(
     data=json.dumps(push_data))
 ```
 
+## Starting feature server in SSL mode
```

Comment:
```suggestion
## Starting the feature server in SSL mode
```

Reply:
I would provide a section above outlining what SSL mode is and why you would use it.